playground/menu.json

```diff
@@ -2,7 +2,7 @@
   "key": "state-machines",
   "uriPath": "state-machines",
   "icon": "RocketIcon",
-  "permissions": ["ViewStates", "ManageStates"],
+  "permissions": ["ManageDeveloperSettings", "ViewDeveloperSettings"],
```

Comment:
Thank god we have integrations tests!

<img width="985" alt="Screenshot 2019-07-11 at 13 25 13" src="https://user-images.githubusercontent.com/1110551/61047594-24fbe300-a3e0-11e9-8c3b-693c20ca0323.png">


Not sure why they haven't been caught up before though...

Reply:
They don't seem to run on PRs aimed at v14? That was my experience anyways... I guess they only run on master would be my wild guess...

Reply:
Yeah and the error only happens since our TS migration as the invariant now runs in render and not as a custom prop-type anymore.

Welcome to my fearfull world of breaking the MC for the last weeks with trying to slowly integrating permissions.

Reply:
btw: I also had to migrate the org for the e2e test account to the new permissions and enable the view,manageDeveloperSettings permission (that's why the tests were still failing)